src/test/java/factories/CheckoutFactory.java

```diff
@@ -23,16 +23,38 @@ public Checkout build() {
         checkout.payment.discountCodes = new ArrayList<>();
 
         checkout.customer = new Customer();
-        //this is set with fake information for now as I just wanted to see if I had achieved the correct format for the JSON
-        checkout.customer.emailAddress = "blah@yahoo.es";
-        checkout.customer.giftOoption = GiftOption.yes;
-        checkout.customer.name = "blah";
+        checkout.customer.emailAddress = new EmailAddressFactory().build();
+        checkout.customer.giftOption = GiftOption.yes;
+        String customerName = new RandomStringFactory().build();
+        checkout.customer.name = customerName;
         checkout.customer.newsletterSignup = true;
-        checkout.customer.phoneNumber = "02086400366";
+        checkout.customer.phoneNumber = new PhoneNumberFactory().build();
         Guest guest = new Guest();
-        guest.name = "yoquese";
+        guest.name = customerName;
```

Comment:
`guest.name = customerName` seems odd; should the customer name and the guest name be the same?